src/Shared/HttpSys/RequestProcessing/NativeRequestContext.cs

```diff
@@ -760,9 +760,12 @@ private uint GetChunksHelper(ref int dataChunkIndex, ref uint dataChunkOffset, b
 
         var info = new Dictionary<int, ReadOnlyMemory<byte>>(count);
 
+        long fixup = (byte*)nativeRequest - (byte*)baseAddress;
```

Comment:
@gfoidl are there guidelines on this? I was just being consistent with usage in this file. Also using var here seems wrong to me since it's non-obvious it's a long to me at least :)